bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
I'm just a user here.

The benefit of a single file is simplicity. In the case of a multilingual site, some pages are LTR and others are RTL. It is a more general approach to have just a single file, it seems to me. Feels more correct to me.

I didn't even know about the `auto` value.

I'd go with a single file, for compatibility and simplicity.

That
> something like direction: auto; in css

that we're waiting for, I think it is the [`:dir` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:dir).